ofl/davidlibre/OFL.txt

```diff
@@ -1,8 +1,8 @@
-Copyright (c) 2014 Monotype Imaging Inc. (licensing@monotype.com). All Rights Reserved. David Hadash is a trademark of Monotype Imaging Inc. Reserved Font Name 'Hadash'. Copyright (c) 2003-2008 SIL International (http://www.sil.org/), with Reserved Font Names 'Gentium' and 'SIL'. Copyright (c) 2015-2016 Meir Sadan (meir@sadan.com).

```

Comment:
@emmamarichal sorry I didn't notice before, the reserved font name shouldn't be removed if part of the font is indeed forked from Gentium. We should have "With reserved font name "Hadash", "Gentium" and 'SIL'." after the copyright string in OFL.txt. But not necessary to include the RFN mention in the font info.